compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -928,9 +934,72 @@ trait Implicits:
           if normalImports.isEmpty then suggestedImports else normalImports
         end hiddenImplicitsAddendum
 
-        msg(userDefined.getOrElse(
-          em"no implicit argument of type $pt was found${location("for")}"))() ++
-        hiddenImplicitsAddendum
+
+        /**
+         * Extracting the message from a method parameter, e.g. in
```

Comment:
The documentation style in the compiler is to start at the first line, and subsequent lines start with two spaces after the `*` to align the message:
```
        /** Extracting the message from a method parameter, e.g. in
         *  ....
         *  ....
         */
```